dev-test/account.test.js

```diff
@@ -0,0 +1,135 @@
+import path from "path"
+import {describe} from "yargs"
+import {
+  emulator,
+  init,
+  getAccountAddress,
+  getContractAddress,
+  deployContractByName,
+  getScriptCode,
+  executeScript,
+  sendTransaction,
+  mintFlow,
+  getFlowBalance,
+  shallRevert,
+  shallResolve,
+  shallPass,
+  shallThrow,
+  createAccount,
+  HashAlgorithm,
+  SignatureAlgorithm,
+} from "../src"
+import {playgroundImport} from "../src/transformers"
+import * as fcl from "@onflow/fcl"
+import {ec as EC} from "elliptic"
+import {validateKeyPair} from "./util/validate-key-pair"
+
+// We need to set timeout for a higher number, because some transactions might take up some time
+jest.setTimeout(10000)
+
+beforeEach(async () => {
+  const basePath = path.resolve(__dirname, "./cadence")
+  await init(basePath)
+  return emulator.start()
+})
+
+afterEach(async () => {
+  emulator.stop()
+})
+
+it("createAccount - should work with name and resolves to correct getAccountAddress", async () => {
+  const addr1 = await createAccount({
+    name: "Billy",
+  })
+
+  const addr2 = await getAccountAddress("Billy")
+
+  expect(addr1).toBe(addr2)
+})
+
+it("createAccount - should work without name and returns address", async () => {
+  const Billy = await createAccount({
+    name: "Billy",
+  })
+
+  expect(Billy).toMatch(/^0x[0-9a-f]{16}$/)
```

Comment:
Would `/^0x[0-9a-f]{16}$/` be nice as a helper? I see it reused a few times and people might like a helper

Reply:
I could make a custom jest matcher for `expect(Foo).toBeAddress()`